### DISCUSSION TITLE
How will the DAMu and EMu interact?

### DISCUSSION THREAD

**Discussion by reguzik** [fieldmuseum/EMu-Documentation#464] [FAQs] [answered]
How will the DAMs and EMu interact?

**Comment by reguzik** [comment]
1) Imaging for Grainger.

View, format, size and number of images is determined by the nature of the digitization project.

The highest quality image(s) should be placed in the appropriate gdi server folder (\gdi).

An appropriately sized web derivative is uploaded to EMu with the appropriate image metadata.

Where the image is of a specimen, object or artifact with a catalogue number, barcode or unique identifier it MUST be linked to a catalogue record(s).

Darwin Core (DWc) metadata will be generated automatically for each catalogue record (http://intranet.fieldmuseum.org/emu/node/5851).

2) What will happen when the DAMs is in place

a scheduled system task (cron job) will collect the images and DWc metadata for new or edited catalogue records and update the appropriate fields in the DAMs.

Selected DWc fields will be searchable NOT editable from the DAMs.

More information on the FMNH DAMs project (https://github.com/fieldmuseum/EMu-Documentation/discussions/1033)

**Comment by rondlg** [comment, marked as answer]
# As of 2023 The Museum's Digital Asset Management System is: NetX
